src/Nav/Nav.css

```diff
@@ -9,6 +9,14 @@ Nav {
   #home-tab, #pokedex-tab, #about-tab{
     border-bottom: 3px solid;
   }
+
+  .content-wrapper {
+    height: 100%;
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+  }
+=======
```

Comment:
mate I think you need to delete this `=======` <-- it's been left over from the file conflicts